projects/chisel.yml

```diff
@@ -30,7 +30,7 @@ primary_contact:
 # Data about active project contributors
 active_committers: https://github.com/chipsalliance/chisel/graphs/contributors
 # Information about project's release methodology and mechanics
-release_methodology: not yet
+release_methodology: As of Chisel 5.0.0 following Semantic Versioning 2.0.0, Chisel3 3.0 - 3.6 followed PVP
```

Comment:
The release process is fully automated on Github, should I also add information about that?